civihr_default_theme/scss/base/_custom.scss

```diff
@@ -628,12 +628,6 @@ h2.pane-title {
 // Dropdowns
 
 .dropdown-menu {
-  padding: 0;
-
-  > li > a {
```

Comment:
as discussed in the call, in order to make the smallest change that would still make things look decent enough, please remove only the `padding` for `.dropdown-menu`, keep the style of `> li > a`

Reply:
also, did you give a quick look around the SSP to find other places where we have `.dropdown-menu` and how this change affects them?

Reply:
I searched through all CiviHR by 'dropdown-menu' in '*.html' files and found out that only these dropdowns will be affected in SSP:

![image](https://user-images.githubusercontent.com/3973243/42284576-dd90c18a-7fa4-11e8-9b74-e6ea107f4d13.png)

![image](https://user-images.githubusercontent.com/3973243/42284618-ff1088cc-7fa4-11e8-9d35-b78b70524e3f.png)

The rest is outside SSP.